.circleci/config.yml

```diff
@@ -93,7 +93,7 @@ jobs:
 
   audit:
     docker:
-      - image: cypress/browsers:node12.0.0-chrome73
+      - image: cypress/browsers:node12.19.0-chrome86-ff82
```

Comment:
I have updated to later version as the new eslint update requires atleast node >= 12.13.0